src/AddIns/Uno.UI.MediaPlayer.Skia.Gtk/Uno.UI.MediaPlayer.Skia.Gtk.csproj

```diff
@@ -0,0 +1,62 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras">
+	<PropertyGroup>
+		<TargetFrameworks>net7.0;net8.0</TargetFrameworks>
```

Comment:
The nuspec only packs `net7.0`. So the `net8.0` build is currently redundant.

@jeromelaban Should we build this for net7.0 only? or adjust the nuspec to pack both net7.0 and net8.0?

Reply:
Yes, let's add net8.0 as well.

Reply:
Adjusted the nuspec, can you check if everything is in order? I adjusted metadata as well (authors, project url etc.)